acceptance/lib/puppet/acceptance/fact_dif.rb

```diff
@@ -0,0 +1,66 @@
+require 'json'
+
+class FactDif
+  EXCLUDE_LIST = %w[os.selinux.enabled
+      networking.mtu fips_enabled mtu_lo
+      mtu_ens192 selinux is_virtual load_averages.1m load_averages.5m load_averages.15m mountpoints./dev.available_bytes
+      system_uptime.seconds system_uptime.days system_uptime.hours uptime_seconds uptime_days uptime_hours
+      identity.privileged identity.gid identity.uid processors.physicalcount processors.count physicalprocessorcount
+      processorcount  memorysize_mb
+      memoryfree_mb swapsize_mb swapfree_mb partitions./dev/mapper/vglocalhost-swap_1.size_bytes
+      partitions./dev/mapper/vglocalhost-root.size_bytes partitions./dev/sda1.size_bytes clientnoop
+      memory.system.available memoryfree system_uptime.uptime uptime facterversion lsbmajdistrelease blockdevices
+      filesystems hypervisors.vmware.version memory.swap.capacity sshfp_ecdsa sshfp_rsa sshfp_ed25519 sshfp_dsa
+      blockdevice_.*_vendor blockdevice_.*_model blockdevice_.*_size mountpoints.* partitions.* operatingsystemrelease
+      os.release.full mtu_.* networking.interfaces.* scope6 disks.* hypervisors.lpar.partition_number.* interfaces.*
+      memory.* processors.speed serialnumber sshdsakey sshrsakey swapfree hypervisors.xen.privileged os.release.minor
+      boardassettag dmi.board.asset_tag boardassettag dmi.board.asset_tag lsbdistrelease lsbminordistrelease
+      processor.* processors.models.* processors.speed bios_release_date bios_vendor bios_version chassisassettag
+      chassistype dmi.* hypervisors.zone.* manufacturer productname virtual zones os.distro.description kernelmajversion
```

Comment:
Would be safer to have the facts that are composed to explicitly contain the `.`?
For example 
- `/dmi.*/` will also exclude `dmix`
- `/dmi\..*/` will only exclude `dmi.` constructs

Not sure if there are any overlapping facts that can get accidentally excluded.

Reply:
Changed this for all facts, except the ones for which we have `.` and  legacy fact notation that start with the same pattern.